### PR TITLE
Update zh-TW.json

### PR DESCRIPTION
**Pull request by caasi**
Update zh-TW.json for #3452

### CODE REVIEW

File: app/javascript/mastodon/locales/zh-TW.json [hunk 1]
```diff
@@ -1,176 +1,176 @@
...
-  "empty_column.home.public_timeline": "the public timeline",
-  "empty_column.notifications": "You don't have any notifications yet. Interact with others to start the conversation.",
-  "empty_column.public": "There is nothing here! Write something publicly, or manually follow users from other instances to fill it up",
-  "follow_request.authorize": "Authorize",
-  "follow_request.reject": "Reject",
+  "account.block": "封鎖 @{name}",
+  "account.block_domain": "隱藏來自 {domain} 的一切",
+  "account.disclaimer": "這使用者來自其他副本，實際數字可能更大。",
+  "account.edit_profile": "編輯用戶資訊",
+  "account.follow": "關注",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/zh-TW.json]: https://github.com/tootsuite/mastodon/issues/3452#issuecomment-305126327

cc @LNDDYL

File: app/javascript/mastodon/locales/zh-TW.json [hunk 2]
```diff
@@ -1,176 +1,176 @@
...
+  "privacy.public.short": "公開貼",
+  "privacy.unlisted.long": "不要貼到公開時間軸",
+  "privacy.unlisted.short": "不列出來",
+  "reply_indicator.cancel": "取消",
+  "report.heading": "新的通報",
+  "report.placeholder": "更多訊息",
+  "report.submit": "送出",
+  "report.target": "通報中",
+  "search.placeholder": "搜尋",
   "search_results.total": "{count, number} {count, plural, one {result} other {results}}",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/zh-TW.json]: FYI: [Japanese case](https://github.com/tootsuite/mastodon/blob/v1.4.1/app/javascript/mastodon/locales/ja.json#L144).

File: app/javascript/mastodon/locales/zh-TW.json (hunk 1 above)
Comment (ykzts) [inline app/javascript/mastodon/locales/zh-TW.json]: > Twitter uses "關注" so I use the same word.

copy from https://github.com/tootsuite/mastodon/issues/3452#issuecomment-305191748

**Review by ykzts — APPROVED** [review]

**Comment by LNDDYL** [comment]
@ykzts Twitter的繁體翻譯也不是很台灣，君不見「應用程序」這樣子的大陸式翻譯………………

File: app/javascript/mastodon/locales/zh-TW.json (hunk 1 above)
Comment (LNDDYL) [inline app/javascript/mastodon/locales/zh-TW.json]: @ykzts Twitter的繁體翻譯也不是很台灣，君不見「應用程序」這樣子的大陸式翻譯………………

**Comment by ykzts** [comment]
@LNDDYL Please use English. Also, there is no point in saying to me. You can make an Issue and discuss it. But it should be done in English. Many people can not understand your words.

**Comment by caasi** [comment]
@LNDDYL Yes. You should open an issue in English and we can discuss there. You can even fire a pull request with your changes like #3519.

**Pull request merged**